pkg/deploy-manager/common.go

```diff
@@ -30,69 +40,114 @@ func (t *DeployManager) CreateNamespace(namespace string) error {
 	return nil
 }
 
+// RemoveAllFinalizers removes all finalizers from every object in the target namespace
+func (t *DeployManager) RemoveAllFinalizers(namespace string) error {
```

Comment:
If things work as expected, why would we need to force Removal of Finalizer from every object in given Namespace?
I see you removed it from script but added it to test code. I'm confused.